content/articles/integrating-chart.js-in-django/index.md

```diff
@@ -0,0 +1,305 @@
+Chart.js is an open-source HTML5 based javascript chart and renders in HTML5 canvas. It basically helps in visualizing data using javascript. 
```

Comment:
In addition to this, can you talk about what other common libraries do we use for charts? How is this better than them? 

2 - 3 lines about this would be enough. This is just to make the intro more descriptive.